### DISCUSSION TITLE
Use img loading="lazy" as a permanent substitute for b-img-lazy

### DISCUSSION THREAD

**Discussion by VividLemon** [cdmoro/bootstrap-vue-3#586] [Ideas]
@mariusa Do you think it would be best to simply add the https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-loading option to imgs, instead of developing a custom solution for b-img-lazy? I believe with `loading="lazy"` being widely supported [here](https://caniuse.com/loading-lazy-attr), it should be a way to easily add that behavior, and give the responsibility to browsers. The original idea was posted https://github.com/cdmoro/bootstrap-vue-3/issues/302#issuecomment-1186492593 from a long dead issue thread

**Comment by VividLemon** [comment]
It should also be noted that https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#browser_compatibility states that every single major browser has `loading` fully supported.

**Reply by mariusa** [reply]
Yes, clearly, let's use `loading` and drop any custom implementation. I would completely drop `b-img-lazy`. I see it's already cut from the docs
https://cdmoro.github.io/bootstrap-vue-3/components/Image.html#component-reference

It's easy to change `b-img-lazy` in an existing project, as it's just a syntax change to `<b-img loading="lazy"`

**Reply by VividLemon** [reply]
Should I add in some extra features though? I believe I can emit an event when the img is loaded, which means that I would be able to allow for a placeholder while loading.

**Reply by mariusa** [reply]
That would be nice, if you prefer.
